lib/public/Comments/IComment.php

```diff
@@ -278,4 +278,20 @@ public function getReferenceId(): ?string;
 	 * @since 19.0.0
 	 */
 	public function setReferenceId(?string $referenceId): IComment;
+
+	/**
+	 * returns the reactions array if exists
+	 *
+	 * @return array e.g. ["👍":1]
+	 */
+	public function getReactions(): array;
+
+	/**
+	 * sets the reactions array
+	 *
+	 * @param array|null $reactions e.g. ["👍":1]
```

Comment:
```suggestion
	 * @param string[]|null $reactions e.g. ["👍":1]
```

and are there legitimate use cases where assigning `null` makes sense? Otherwise I would limit the API to assignments of non-null array.